vendor/assets/javascripts/initializers.js

```diff
@@ -71,7 +71,7 @@ $(document).on('turbolinks:load', function() {
   $('.js-featured-image-table-element').each(function() {
     var imageUrl = $(this).data('imageUrl');
     $(this).find('input').on('click', function(event) {
-      $('.js-featured-image').attr('src', imageUrl);
+      $('.js-featured-image').attr('src', imageUrl).removeClass('uk-hidden');
```

Comment:
This is fine for now but we will need to decouple this from the Featured Image case so that it works in any image upload situation.